### PR TITLE
test: test case cleanups

### PR DESCRIPTION
**Pull request by mxschmitt**
Didn't run the tests locally, we should see if they are passing tho no the CI.

### CODE REVIEW

File: src/ContosoTraders.Ui.Website/tests/api.cart.spec.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const PRODUCTQUANTITY = 1;
 
 // SETUP: Create a new cart
 test.beforeAll(async ({ request }) => {
-    const newCart = await request.post(`${process.env.REACT_APP_APIURLSHOPPINGCART}/ShoppingCart`, {
```
Comment (MarcusFelling) [inline src/ContosoTraders.Ui.Website/tests/api.cart.spec.ts:13]: this is separate from the baseURL (it's for the ACA shopping cart API), so using a relative path here won't work.

Comment (MarcusFelling) [reply src/ContosoTraders.Ui.Website/tests/api.cart.spec.ts:13]: see test failures here: https://github.com/microsoft/contosotraders-cloudtesting/actions/runs/4746674582/jobs/8430895228

Comment (mxschmitt) [reply src/ContosoTraders.Ui.Website/tests/api.cart.spec.ts:13]: changed it to test.use which should work and seems prettier here.

Comment (MarcusFelling) [reply src/ContosoTraders.Ui.Website/tests/api.cart.spec.ts:13]: great idea!

**Review by MarcusFelling — APPROVED** [review]

**Pull request merged**